### PR TITLE
Forbid adding schema properties if they existed before

### PR DESCRIPTION
**Pull request by wenhaocs**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
Case 1 in https://github.com/vesoft-inc/nebula/issues/2671
#### Description:
When we change schema, the existing data will not be changed. In case 1 above, if a user adds a column/property back (with the same name, but has different attributes) after dropping it, querying the existing data will show the properties under the old schema, not matching the new schema. Per discussion, we just forbid adding column/property back with the same name.

## Checklist:
Tests:
- [x] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> Adding column/property back with the same name is forbidden.

### CODE REVIEW

File: tests/query/stateless/test_schema.py [hunk 1]
```diff
@@ -124,23 +124,15 @@ def test_alter_tag_succeed(self):
...
-        self.check_result(resp, expect)
-
-        resp = self.execute('DESC TAG student')
-        self.check_resp_succeeded(resp)
-        expect = [['name', 'string', 'YES', T_EMPTY, T_EMPTY],
-                  ['email', 'string', 'YES', T_EMPTY, T_EMPTY],
-                  ['birthday', 'timestamp', 'YES', T_EMPTY, T_EMPTY],
-                  ['age', 'string', 'YES', T_EMPTY, T_EMPTY]]
+                  ['age_str', 'string', 'YES', T_EMPTY, T_EMPTY]]
         self.check_result(resp, expect)
```
Comment (wenhaocs) [inline tests/query/stateless/test_schema.py:136]: These are redundant.

File: src/common/utils/MetaKeyUtils.cpp [hunk 2]
```diff
@@ -573,16 +573,17 @@ SchemaVer MetaKeyUtils::parseEdgeVersion(folly::StringPiece key) {
          *reinterpret_cast<const SchemaVer*>(key.begin() + offset);
 }
 
-SchemaVer MetaKeyUtils::getLatestEdgeScheInfo(kvstore::KVIterator* iter, folly::StringPiece& val) {
+SchemaVer MetaKeyUtils::getLatestEdgeScheInfo(
+    kvstore::KVIterator* iter, std::unordered_map<SchemaVer, folly::StringPiece>& schemasRaw) {
```
Comment (critical27) [inline src/common/utils/MetaKeyUtils.cpp:577]: Maybe rename to schemaHistory or something else?

**Review by critical27 — APPROVED** [review]
Good job~

**Review by SuperYoko — APPROVED** [review]

**Comment by abby-cyber** [comment]
https://github.com/vesoft-inc/nebula-docs-cn/pull/2496

**Pull request merged**